option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message HeroEmbattleRequest
{
 required uint32 pos = 1;
 repeated U64Type guid = 2;
}
